pkg/collectors/addon.go

```diff
@@ -88,12 +88,21 @@ func (cc *AddonCollector) collectAddon(ch chan<- prometheus.Metric, addon *kuber
 
 	clusterName := addon.OwnerReferences[0].Name
 
+	resourcesCreated := "0"
+
+	for _, cond := range addon.Status.Conditions {
+		if cond.Type == kubermaticv1.AddonResourcesCreated {
+			resourcesCreated = "1"
+		}
+	}
+
```

Comment:
Not sure if adding this as a label to the creation timestamp metric is the optimal approach. What about having a dedicated metric for this?

Reply:
Could you elaborate more on this?

Reply:
Instead of extensing the `kubermatic_addon_created` metric with a (somewhat) unrelated label, why not create a new like `kubermatic_addon_has_resources`?